### DISCUSSION TITLE
ALL_CAPS private const field bypasses the rules

### DISCUSSION THREAD

**Discussion by TheToughGuy** [RehanSaeed/EditorConfig#53] [Q&A] [answered]
Hi,

I'm having a play with editorConfig and applying the PascalCase rule for private const field like

`private const string SESSIONDB_CONTEXT_JSON_DATA_PATH = "./Data/Tenant.json";`

but I do not get a warning, I assume the following section should be dealing with this?

# All constant fields must be PascalCase
# https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1303.md
dotnet_naming_symbols.stylecop_constant_fields_group.applicable_accessibilities = public, internal, protected_internal, protected, private_protected, private
dotnet_naming_symbols.stylecop_constant_fields_group.required_modifiers         = const
dotnet_naming_symbols.stylecop_constant_fields_group.applicable_kinds           = field
dotnet_naming_rule.stylecop_constant_fields_must_be_pascal_case_rule.symbols    = stylecop_constant_fields_group
dotnet_naming_rule.stylecop_constant_fields_must_be_pascal_case_rule.style      = pascal_case_style
dotnet_naming_rule.stylecop_constant_fields_must_be_pascal_case_rule.severity   = warning

**Comment by RehanSaeed** [comment, marked as answer]
Just tested this and I can see that VS 16.9 doesn't show a warning. This used to work but not sure when it stopped working. I tried a blank .editorconfig file with nothing but these rules in it and I got nothing.

Just ran through the docs on these custom rules again and it seems like we are doing the right thing. We should probably check for an issue or raise one in the Roslyn GitHub repo.

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/naming-rules

**Reply by TheToughGuy** [reply]
Thanks for such a quick response, I have duplicated into the project

https://github.com/dotnet/roslyn/issues/52865
